roles/download/defaults/main.yml

```diff
@@ -411,7 +420,6 @@ kubeadm_checksums:
     v1.17.2: 33a1d8e3cea2bdbb9fa9cb257c516289ee50d957fcb6d7b35919f5f0e6ca2f41
     v1.17.1: 11bd31833dab9adb5b53398772dd1582264c3d1757cb3395e691d6a7379081ec
     v1.17.0: 0d8443f50fb7caab2e5e7e53f9dc56d5ffe55f021ec061f2e2bcba0481df5a48
-
```

Comment:
We don't need to remove this blank line because the other block also has it like the original line 441.

Reply:
Got it! Thought I added that by mistake I'll add it back

Reply:
https://github.com/kubernetes-sigs/kubespray/pull/7012/commits/067183ac124a18572d56ab7285b8fba91403d3e2